spec/features/card_pages/polling_spec.rb

```diff
@@ -15,7 +15,7 @@
       end
 
       it 'adds a link' do
-        expect(page).to have_selector "#card_#{@new_card.id}_link"
+        expect(page).to have_selector "#card_#{@new_card.id}_link", visible: false
```

Comment:
These changes are needed because of load times?

Reply:
This is needed becuase the sidebar is closed in some tests. I figuered this was better than checking the sidebars state and toggling it conditionally for every test